### PR TITLE
Fixes some problems in xcode 8 / swift 3

### PR DESCRIPTION
**Pull request by azt3k**

### CODE REVIEW

**Comment by azt3k** [comment]
Looks like you've applied the same changes in master

**Comment by williamngan** [comment]
Oh wait, sorry, didn't see your pull request until now.

There're still some good stuff in your pull request, so if you want to reopen it I can merge it.

I added a comment in setRepeatCount.

Thanks!

File: Sprite/Sprite.swift [hunk 1]
```diff
@@ -48,10 +48,9 @@ public class Sprite: UIView {
   public var _repeatCount: Int = 0;
   
   public func setRepeatCount( n: Int ) {
-
-    if n >= 0 && n != _repeatCount {
-        _repeatCount = n
-        sprite!.animationRepeatCount = n
+    print("repeat", n)
```
Comment (williamngan) [inline Sprite/Sprite.swift:51]: remove print statement

File: Sprite/Sprite.swift [hunk 2]
```diff
@@ -87,14 +86,14 @@ public class Sprite: UIView {
...
   
   // bridge for React property setter
   public func setAnimated( shouldPlay:Bool ) {
     _animated = shouldPlay
     
-    if _animated != sprite!.isAnimating() {
+    if _animated != sprite!.isAnimating {
       if shouldPlay {
         sprite!.animationImages = seq
         sprite!.animationRepeatCount = _repeatCount
```
Comment (williamngan) [inline Sprite/Sprite.swift:99]: This still uses the `_repeatCount`, which you removed above in `setRepeatCount()`. Can we remove this line safely?

File: Sprite/Sprite.swift [hunk 3]
```diff
@@ -87,14 +86,14 @@ public class Sprite: UIView {
   private var _animated: Bool = false
   
   public func animate( shouldPlay: Bool ) {
-    setAnimated(shouldPlay)
+    setAnimated(shouldPlay: shouldPlay)
```
Comment (williamngan) [inline Sprite/Sprite.swift:89]: Sorry I'm not familiar with this syntax... what does it do? 😅

File: Sprite/Sprite.swift [hunk 4]
```diff
@@ -69,7 +68,7 @@ public class Sprite: UIView {
   public func setImageNumber( n: Int ) {
     
     _imageNumber = n
-    if !sprite!.isAnimating() && seq != nil {
+    if !sprite!.isAnimating && seq != nil {
```
Comment (williamngan) [inline Sprite/Sprite.swift:71]: If you drop the (), I think it'll return the function itself instead of the Bool value. Right?

**Pull request closed without merging**